README.md

```diff
@@ -36,6 +36,11 @@ render(<Main />, document.body);
 
 If there is an error rendering the destination route, a 404 will be displayed.
 
+#### Caveats
+
+Because the `path` and `default` props are used by the router, it's better to avoid
+same props in your components.
```

Comment:
```suggestion
Because the `path` and `default` props are used by the router, it's best to avoid using those props for your component(s) as they will conflict.
```